core/modules/sink.h

```diff
@@ -5,6 +5,8 @@
 
 class Sink final : public Module {
  public:
+  Sink() { max_allowed_workers_ = Worker::kMaxWorkers; }
```

Comment:
Last commit should have fixed the concern with queue. Will walk through all the modules in general.